test/integration/sysched_test.go

```diff
@@ -0,0 +1,333 @@
+			Namespace: "default",
+		},
+		Spec: spo.SeccompProfileSpec{
+			Architectures: []spo.Arch{"SCMP_ARCH_X86_64"},
+			DefaultAction: "SCMP_ACT_LOG",
+			Syscalls: []*spo.Syscall{{
+				Action: "SCMP_ACT_ALLOW",
+				Names:  []string{"clone", "socket", "getuid", "setrlimit", "nanosleep", "sendto", "setuid", "getpgrp", "mkdir", "getegid", "getsockname", "clock_gettime", "prctl", "epoll_pwait", "futex", "link", "ftruncate", "access", "gettimeofday", "select", "getsockopt", "mmap", "write", "connect", "capget", "chmod", "arch_prctl", "wait4", "brk", "stat", "getrlimit"},
+			},
+			},
+		},
+	}
+)
+
+func SPOCreate(client *spoclient.SPOV1Alpha1Client, profile *spo.SeccompProfile, opts metav1.CreateOptions) (*spo.SeccompProfile, error) {
+	result := spo.SeccompProfile{}
+	err := client.RestClient.
+		Post().
+		Namespace("default").
+		Resource("seccompprofiles").
+		VersionedParams(&opts, scheme.ParameterCodec).
+		Body(profile).
+		Do(context.TODO()).
+		Into(&result)
+
+	return &result, err
+}
+
+func SPOGet(client *spoclient.SPOV1Alpha1Client, name string, ns string, opts metav1.GetOptions) (*spo.SeccompProfile, error) {
+	result := spo.SeccompProfile{}
+	err := client.RestClient.
+		Get().
+		Namespace("default").
+		Resource("seccompprofiles").
+		Name(name).
+		VersionedParams(&opts, scheme.ParameterCodec).
+		Do(context.TODO()).
+		Into(&result)
+
+	return &result, err
+}
+
+func TestSyschedPlugin(t *testing.T) {
+	testCtx := &testContext{}
+	testCtx.Ctx, testCtx.CancelFn = context.WithCancel(context.Background())
+
+	cs := kubernetes.NewForConfigOrDie(globalKubeConfig)
+
+	spo.AddToScheme(scheme.Scheme)
+	extClient, err := spoclient.NewForConfig(globalKubeConfig)
+	if err != nil {
+		t.Fatal(err)
+	}
+	testCtx.ClientSet = cs
+	testCtx.KubeConfig = globalKubeConfig
+
+	if err := wait.Poll(100*time.Millisecond, 3*time.Second, func() (done bool, err error) {
+		groupList, _, err := cs.ServerGroupsAndResources()
+		if err != nil {
+			return false, nil
+		}
+		for _, group := range groupList {
+			if group.Name == "security-profiles-operator.x-k8s.io" {
+				t.Log("The CRD is ready to serve")
+				return true, nil
+			}
+		}
+		return false, nil
+	}); err != nil {
+		t.Fatalf("Timed out waiting for CRD to be ready: %v", err)
+	}
+
+	// Create the Seccomp Profile CRs
+	_, err = SPOCreate(extClient, &fullseccompSPOCR, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = SPOCreate(extClient, &badSPOCR, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = SPOCreate(extClient, &good1SPOCR, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = SPOCreate(extClient, &good2SPOCR, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	cfg, err := util.NewDefaultSchedulerComponentConfig()
+	if err != nil {
+		t.Fatal(err)
+	}
+	cfg.Profiles[0].Plugins.Score = schedapi.PluginSet{
+		Enabled: []schedapi.Plugin{{Name: sysched.Name}},
+		//Disabled: []schedapi.Plugin{{Name: "*"}},
```

Comment:
You should be disabling it. Otherwise score of other plugins may impact the eventual result.